src/main/java/org/beanpod/switchboard/config/WebMvcConfiguration.java

```diff
@@ -13,7 +13,8 @@ public WebMvcConfigurer webMvcConfigurer() {
     return new WebMvcConfigurer() {
       @Override
       public void addCorsMappings(CorsRegistry registry) {
-        registry.addMapping("/**").allowedMethods("GET", "POST", "PUT", "DELETE");
+        registry.addMapping("/**")
+            .allowedMethods("*");
```

Comment:
Suggested change to pass Google Java Format.
```suggestion
        registry.addMapping("/**").allowedMethods("*");
```

Reply:
super!